plugins/server-action/index.ts

```
@@ -0,0 +1,114 @@
+import { generateRandomString } from "../../features/utils";
+import { builder } from "../../internal/server/build";
+import { type BunextPlugin } from "../types";
+import { normalize } from "path";
+
+async function ServerActionToTag(moduleContent: Record<string, unknown>) {
+  return Object.entries(moduleContent)
+    .filter(([ex, _]) => ex.startsWith("Server"))
+    .reduce(
+      (a, [ex, _]) => ({
+        ...a,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)